### PR TITLE
Restructuring README

### PR DESCRIPTION
**Pull request by CsatariGergely**
- Focus more on that this is only the source of the documentation
- Replace GitHub link with docs.nephio.org ones
- Add Netlify build status

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 # Nephio documentation
 
-This repository contains the documentation for the Nephio project. 
+This repository contains the source filed for the documentation for the [Nephio](https://nephio.org/) project.
```
Comment (liamfallon) [inline README.md:3]:
```suggestion
This repository contains the source files for the documentation for the [Nephio](https://nephio.org/) project. 
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+This repository contains the source filed for the documentation for the [Nephio](https://nephio.org/) project. 
+The documentation is served from [docs.nephio.org](https://docs.nephio.org/) and the status of our Netlify build is
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/9a7a49cd-9710-49c7-bd97-1dfb2272717c/deploy-status)](https://app.netlify.com/sites/nephio/deploys)
 
 ## Status of the documentation
 
-We are in the middle fo moving the documentation from a GitHub based documentation to a Hugo / Docsy based documentation
-site. The content of this repo (especially this) branch will be a bit chaotic during this time. 
+In this release the documentation was moved from a GitHub based documentation to a Hugo / Docsy based site. We are still working on the restructuring the content and finalizing the look and feel of the page.
```
Comment (liamfallon) [inline README.md:10]:
```suggestion
In this release, a a Hugo / Docsy based documentation site was introduced for the Nephio documentation. The source documentation remains the Markdown files in the Github repo and the Hugo / Docsy documentation is generated from the Markdown files. We are still working on the restructuring the content and finalizing the look and feel of the page. 
```

**Comment by liamfallon** [comment]
@CsatariGergely just wondering if you want to take a look at my comments?

File: README.md (hunk 2 above)
Comment (electrocucaracha) [reply README.md:10]: I would be more specific about the release
```suggestion
In R2 release, a Hugo / Docsy based documentation site was introduced for the Nephio documentation. Hugo / Docsy uses the Markdown files hosted in the Github repo to generate the documentation website. We are still working on the restructuring the content and finalizing the look and feel of the website.
```

**Pull request merged**